src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -119,3 +119,10 @@ def devices(self, hostname):
     def smart(self, hostname):
         # type: (str) -> dict
         return CephService.get_smart_data_by_host(hostname)
+
+    @RESTController.Resource('GET')
+    @raise_if_no_orchestrator
+    def daemons(self, hostname):
+        orch = OrchClient.instance()
```

Comment:
Hmm..Should be `# type (str) -> List[dict]`. Since cherrypy doesn't know how to serialize `DaemonDescription` instance. I add typing to the changes in this PR.